sim/verilator-uhdm/meta.yaml

```diff
@@ -11,29 +13,31 @@ source:
 
 build:
   number: {{ environ.get('DATE_NUM') }}
-  string: {{ environ.get('DATE_STR') }}
+  string: {{ environ.get('DATE_STR') }}_{{ py_suffix }}
   script_env:
     - CI
-    - USE_SYSTEM_GCC_VERSION
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - python {{ python }}
+    # These version restrictions make the dependencies resolvable with
+    # packages from the `defaults` channel (libgcc-ng/libstdcxx-ng).
+    - {{ compiler('c') }} <9
+    - {{ compiler('cxx') }} <9
```

Comment:
Drop the comment and you can also remove the restrictions. I can see that the package built requires `libunwind` to be installed so it won't work without `conda-forge` anyway.

Reply:
Also, please remove `unused` from the last commit title. It definitely was used but if the package requires `conda-forge` to be installed then it shouldn't be there.